extension/sumologicextension/extension.go

```diff
@@ -715,13 +715,17 @@ var sumoAppProcesses = map[string]string{
 func filteredProcessList() ([]string, error) {
 	var pl []string
 
-	p, err := ps.Processes()
+	p, err := process.Processes()
 	if err != nil {
 		return pl, err
 	}
 
 	for _, v := range p {
-		e := strings.ToLower(v.Executable())
+        e, err := v.Name()
```

Comment:
Why are we using the process name now, when in the past we were using the executable? I'd prefer if we replace the library without any semantic changes first.

Reply:
From documentation it looks like this is for the same underlying value, or at least, the closest available (from what I can tell).

[`go-ps` doc reference](https://pkg.go.dev/github.com/mitchellh/go-ps#Process) for `Executable()`:
```
Executable name running this process.
```
[`gopsutil` doc reference](https://pkg.go.dev/github.com/shirou/gopsutil/v3@v3.24.5/process#Process.Name) for `Name()`:
```
Name returns name of the process.
```
@harshshahsumo Can you confirm that the same value is being returned here?

Reply:
I thought we could use https://pkg.go.dev/github.com/shirou/gopsutil/v3@v3.24.5/process#Process.Exe. It returns a full path to the executable, but it's easy to extract just the binary name. I tried to look through the implementation of https://pkg.go.dev/github.com/shirou/gopsutil/v3@v3.24.5/process#Process.Name, and it was complex enough that I'm not confident it will always give the same result.

Reply:
Thanks for investigating! Sounds good to me. 

I'll add a label to this PR to make sure the CI/CD actions run on Windows to make sure any path logic works properly there as well 👍